Flipper/Packages/Core/Sources/Archive/Item/ArchiveItem+Property.swift

```diff
@@ -3,6 +3,12 @@ extension ArchiveItem {
         public let key: String
         public var value: String
         public var description: [String] = []
+
+        public init(key: String, value: String, description: [String] = []) {
+            self.key = key
```

Comment:
Need constructor? Because in UI module, we can not create Property with internal level  protection